src/useRestActions.ts

```diff
@@ -66,7 +66,7 @@ const useRestActions = <Schema>(conf?: Partial<Config>, initialData?: Schema[])
     }
   }
   const update = useCreateAction<Schema>(handleSuccessUpdate as HandleSuccess<Schema>, shouldMakeRequest(updateConfig) ? "PUT" : undefined, updateConfig, actionCreators, dispatch)
-  const updateStable = useCallback(update, [])
```

Comment:
I think we're fooling the here.
We're making a 'stable' version of update, where the linter is not able to see which dependencies it has.

Dependencies are:
`handleSuccessUpdate, updateConfig, actionCreators, dispatch`

`handleSuccessUpdate` should be a useCallback itself, with its own dependencies. (state)

After an update the `handleSuccessUpdate` is still referencing to an empty version of state, because updateStable is memoized at the beginning, but is never renewed.